parser_library/src/workspaces/library_local.cpp

```diff
@@ -101,6 +101,8 @@ std::vector<std::string> library_local::list_files()
     return result;
 }
 
+std::string library_local::refresh_url_prefix() const { return m_lib_loc.get_uri(); }
```

Comment:
Why don't return just the `resource_location` itself instead of `std::string`? Due to performance reasons during the search later?

Reply:
I don't think there would be a perf impact, but what the library impl returns here is not required to be a valid url.